sdk/src/test/scala/com/horizen/vrf/VrfFunctionsTest.scala

```diff
@@ -0,0 +1,67 @@
+package com.horizen.vrf
+
+import java.util
+
+import com.horizen.vrf.VrfFunctions.{KeyType, ProofType}
+import org.junit.Assert.{assertEquals, assertFalse, assertNotEquals, assertTrue}
+import org.junit.Test
+
+import scala.util.Random
+
+
+class VrfFunctionsTest {
+  val keys: util.EnumMap[KeyType, Array[Byte]] = VrfLoader.vrfFunctions.generatePublicAndSecretKeys(1.toString.getBytes())
+  val secretBytes: Array[Byte] = keys.get(KeyType.SECRET)
+  val publicBytes: Array[Byte] = keys.get(KeyType.PUBLIC)
+  val message: Array[Byte] = "Very secret message!".getBytes
+  val vrfProofBytes: Array[Byte] = VrfLoader.vrfFunctions.createVrfProof(secretBytes, publicBytes, message).get(ProofType.VRF_PROOF)
+  val vrfProofCheck: Boolean = VrfLoader.vrfFunctions.verifyProof(message, publicBytes, vrfProofBytes)
+  val vrfOutputBytes: Array[Byte] = VrfLoader.vrfFunctions.vrfProofToVrfOutput(publicBytes, message, vrfProofBytes).get()
+
+  @Test
+  def sanityCheck(): Unit = {
+    assertNotEquals(vrfProofBytes.deep, vrfOutputBytes.deep)
+    assertTrue(VrfLoader.vrfFunctions.publicKeyIsValid(publicBytes))
+    assertTrue(vrfProofCheck)
+    assertTrue(vrfOutputBytes.nonEmpty)
+  }
+
+
+  @Test
+  def determinismCheck(): Unit = {
```

Comment:
can be "vrfOutputDeterminismCheck".
I would suggest to add the comment that proof is not deterministic instead of commenting the check in line 40 or even add another test and mark in `@Ignored`